packages/react/src/LogoSuite/LogoSuite.tsx

```diff
@@ -153,7 +153,7 @@ type LogoSuiteLogoBarProps = BaseProps<HTMLDivElement> & {
   /**
    * The speed of the marquee effect
    */
-  marqueeSpeed?: 'slow' | 'normal'
+  marqueeSpeed?: 'slow' | 'normal' | 'stop'
```

Comment:
How about `idle` for this new option?

'Stop' feels like an action, whereas the other existing values allude to the current state of the speed. Idle is also used elsewhere in the system, so consistent verbiage.